i18n/locales/es.ts

```diff
@@ -0,0 +1,376 @@
+        'A <a href="https://bitcoinops.org/en/topics/block-explorers/" target="_blank" rel="noreferrer">block explorer</a> is a useful tool to quickly look up information about bitcoin transactions.',
+      tooltip_hex:
+        'Abreviatura de hexadecimal, un sistema numérico que representa números utilizando una base de 16.',
+      view_block_0: 'Ver bloque 0',
+      placeholder: 'Pega el valor que has encontrado aquí',
+    },
+
+    genesis_three: {
+      title: 'Génesis',
+      heading: 'Descifremos el mensaje',
+      paragraph_one:
+        'El mensaje que encontraste estaba codificado en un formato llamado HEX. Ahora ejecutaremos un comando para convertirlo en ASCII, que podremos leer',
+      paragraph_two:
+        'Copia y pega el comando de abajo en el Terminal de la derecha y pulsa "Enter"',
+      terminal_challenge_lines: 'Introduzca sus comandos aquí y pulse Intro...\n\n La variable $scriptSigHex ya está definida para usted.\n\n var $scriptSigHex = "04fff...e6b73"',
+      waiting_for_input: 'Esperando a que escribas y ejecutes el script...',
+      éxito: 'Buen trabajo! El mensaje descifrado hace referencia a la portada de <Link href="https://en.bitcoin.it/wiki/Genesis_block" className="underline">The Times</Link> del 3 de enero de 2009, el mismo día que Satoshi minó el bloque génesis. ¿No es genial? Este mensaje también nos da una idea de su motivación para crear bitcoin,',
+    },
+
+    genesis_four: {
+      title: 'Génesis',
+      subtitle: '¡Felicidades! ¡Has completado el primer reto!',
+      paragraph_one:
+        'Encontraste el mensaje secreto que Satoshi Nakamoto incrustó en el bloque génesis. Es hora de ampliar lo que acabas de aprender. Descifra una pista importante sobre la historia en el siguiente desafío.',
+    },
+
+    transacting_one: {
+      title: 'Transaccionando',
+      heading: '¿Qué hay en una transacción?',
+      paragraph_one:
+        'Las transacciones tienen dos partes: entradas y salidas. En el ejercicio anterior desciframos un mensaje secreto que se encontraba dentro de la entrada de una transacción. Esta vez descifraremos un mensaje que pertenece a la parte de salida',
+      paragraph_two:
+        'Para la siguiente transacción, vamos a identificar la salida de tipo OP_RETURN.',
+    },
+
+    transacting_two: {
+      title: 'Transaccionando',
+      heading: 'OP_RETURN',
+      paragraph_one:
+        'Hay otra forma de ocultar mensajes secretos en las transacciones. Bitcoin tiene un tipo especial de código llamado OP_RETURN que permite a los usuarios adjuntar mensajes a las salidas de las transacciones. Veamos si podemos encontrar uno',
+      paragraph_two:
+        '1. Haga clic en <Link href="https://blockstream.info/tx/ff9148605a772a51cba39004df5fb042d40515967a3e38ff5294cfd017c452a9?expand" className="underline">aquí</Link> para ver una transacción específica.',
+      paragraph_three:
+        '2. Abra los detalles y busque la parte que es de tipo "OP_RETURN".',
```

Comment:
Using "usted" instead of "tú" (which is what you have mostly used):

```suggestion
        '2. Abre los detalles y busca la parte que es de tipo "OP_RETURN".',
```